deploy/microshift-bootc/README.md

```diff
@@ -0,0 +1,420 @@
+| `make bootc-run` | Run MicroShift in a container |
+| `make bootc-stop` | Stop the running container |
+| `make bootc-rm` | Remove container and clean up LVM resources |
+| `make bootc-sh` | Open shell in container |
+| `make bootc-reload-app` | Reload config service without rebuild (dev mode) |
+| `make build-image` | Create bootable QCOW2 image |
+| `make bootc-push` | Push image to registry |
+| `make clean` | Clean up images, artifacts, and LVM disk |
+
+## Files
+
+| File | Description |
+|------|-------------|
+| `Containerfile` | Container build definition |
+| `config.toml` | Bootc image builder configuration |
+| `run-microshift.sh` | Container startup script |
+| `kustomization.yaml` | Kubernetes manifests configuration |
+| `config-svc/app.py` | Configuration web UI service |
+| `config-svc/config-svc.service` | Systemd service definition |
+
+## Network Configuration
+
+### Hostname Resolution
+
+The system uses `nip.io` for automatic DNS resolution:
+- Default: `jumpstarter.<host-ip>.nip.io`
+- Example: `jumpstarter.10.0.2.2.nip.io` resolves to `10.0.2.2`
+
+### Firewall Ports
+
+| Port | Service | Description |
+|------|---------|-------------|
+| 80 | HTTP | MicroShift ingress |
+| 443 | HTTPS | MicroShift API and ingress |
+| 8880 | Config UI | Web configuration interface |
+| 6443 | API Server | Kubernetes API (internal) |
+
+## Security Notes
+
+⚠️ **Important Security Considerations:**
+
+1. **Default Password:** The system ships with `root:jumpstarter` as the default password
+   - **Console login:** You will be forced to change the password on first SSH/console login
+   - **Web UI:** You must change the password before accessing the configuration interface
+2. **TLS Certificates:** MicroShift uses self-signed certs by default
+3. **Privileged Container:** Required for systemd, LVM, and networking
+4. **Authentication:** Web UI uses PAM authentication with root credentials
+5. **Production Use:** Consider additional hardening for production deployments
+
+## Development Workflow
+
+Typical development cycle:
+
+```bash
+# 1. Make changes to code/configuration
+vim config-svc/app.py
+
+# 2. Quick reload (no rebuild needed)
+make bootc-reload-app
+
+# 3. Access and test
+curl http://localhost:8880
+
+# 4. Check logs if issues
+make bootc-sh
+journalctl -u config-svc -f
+
+# 5. For major changes, do full rebuild
+make bootc-rm bootc-build bootc-run
+```
+
+## Production Deployment
+
+1. **Build QCOW2 image:**
+   ```bash
+   make build-image
+   ```
+
+2. **Copy image to target system:**
+   ```bash
+   scp output/qcow2/disk.qcow2 target-host:/var/lib/libvirt/images/
+   ```
+
+3. **Create VM or write to disk:**
+   ```bash
+   # For VM
+   virt-install --name jumpstarter \
+       --memory 4096 \
+       --vcpus 2 \
+       --disk path=/var/lib/libvirt/images/disk.qcow2 \
+       --import \
+       --os-variant fedora39
+   
+   # For bare metal
+   dd if=output/qcow2/disk.qcow2 of=/dev/sdX bs=4M status=progress
```

Comment:
```suggestio
```

you cannot flash a qcow2 to disk! 

We need a section at the start explaining how to install into a baremetal device, by creating a raw and flashing into storage, or by using an iso.